mmf/datasets/builders/coco/masked_dataset.py

```diff
@@ -20,6 +20,9 @@ def load_item(self, idx):
         if self._use_features is True:
             features = self.features_db[idx]
 
+            if self.config.get("use_image_bbox_position", False):
```

Comment:
I find it little conflicting that we have two options doing same thing. If bbox processor is defined then it means we want to use it, no? Do we really need extra option?